estimator/transport_co2/model.py

```diff
@@ -60,6 +58,21 @@ def __init__(self, avg_co2_per_vehicle_km: float, avg_occupancy: float):
     SCOOTER = (86.4, 1.2)
     BUS = (863, 12.7)
 
+    # Additional OTP modes
+    WALK = (0, 1)
+    BICYCLE = (0, 1)
+    CAR = (mean([SMALL_CAR[0], LARGE_CAR[0]]), 1.5)
+    TRAM = LIGHT_RAIL
+    SUBWAY = LIGHT_RAIL
+    RAIL = LIGHT_RAIL
+    FERRY = (15014.4, 184)
+    CABLE_CAR = LIGHT_RAIL
+    GONDOLA = LIGHT_RAIL
+    FUNICULAR = LIGHT_RAIL
```

Comment:
Everything's a light rail these days, huh? 😉 Hopefully we'll get some better values for the next iteration. 😎

Reply:
Lol. I'm not sure what else to do here. I could research some values for the other modes. I hadn't even heard if a funicular before this, but it sounds a bit like fun-tacular.

Reply:
@MarkusShepherd I have made a couple of improvements here. Specifically, I added a calculation for SUBWAY (metro) including documenting the data source.